### PR TITLE
Rebase cAdvisor to the new libcontainer.

### LINKED ISSUE #633: cAdvisor does not support Docker 1.6
Working on rebasing to the new libcontainer and fix this (will also make sure to be backwards compatible.)

### PR DESCRIPTION
**Pull request by vmarmol**
This maintains backwards compatibility with older Dockers while still providing support for Docker 1.6.

Fixes #633.

### CODE REVIEW

File: container/libcontainer/helpers.go [hunk 1]
```diff
@@ -16,13 +16,18 @@ package libcontainer
...
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/docker/libcontainer"
 	"github.com/docker/libcontainer/cgroups"
-	cgroupfs "github.com/docker/libcontainer/cgroups/fs"
-	"github.com/docker/libcontainer/network"
+	//cgroupfs "github.com/docker/libcontainer/cgroups/fs"
+	//"github.com/docker/libcontainer/network"
```
Comment (rjnagal) [inline container/libcontainer/helpers.go]: remove commented.

File: Godeps/Godeps.json [hunk 2]
```diff
@@ -155,18 +159,6 @@
...
-		},
-		{
-			"ImportPath": "golang.org/x/oauth2/jwt",
-			"Rev": "ca8a464d23d55afd32571475db223e065ffd8a52"
-		},
-		{
-			"ImportPath": "golang.org/x/oauth2/jws",
-			"Rev": "ca8a464d23d55afd32571475db223e065ffd8a52"
-		},
-		{
```
Comment (rjnagal) [inline Godeps/Godeps.json]: Removing these is causing the build to fail.

Comment (vmarmol) [inline Godeps/Godeps.json]: Hmm wonder why Godep did that...added back.

File: container/libcontainer/helpers.go (hunk 1 above)
Comment (vmarmol) [inline container/libcontainer/helpers.go]: Thanks, forgot about those :)

**Comment by vmarmol** [comment]
Network stats are failing on Ubuntu. Will take a look later today when I have a bit more time.

**Comment by rjnagal** [comment]
@vmarmol Did you fix the network stats already?

**Comment by vmarmol** [comment]
Not yet, will get to it today I promise :)

**Comment by rjnagal** [comment]
No rush, I was just wondering why the tests are passing.

**Comment by vmarmol** [comment]
Tests are passing (not sure why the initial failures, I haven't been able to reproduce them). However, Jenkins is failing due to the OOM test. Did that pass for the previous PR?

**Comment by vmarmol** [comment]
Jenkins passes: http://104.154.52.74/job/cadvisor-e2e/778/

**Comment by vmarmol** [comment]
PTAL @rjnagal :)

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**